docs/content/en/docs/reference/clusterspec/optional/registrymirror.md

```diff
@@ -78,13 +78,21 @@ export REGISTRY_USERNAME=<username>
 export REGISTRY_PASSWORD=<password>
 ```
 
+Download and import EKS Anywhere images:
 ```bash
 eksctl anywhere download images -o eks-anywhere-images.tar
 docker login https://<private registry endpoint>
 ...
 eksctl anywhere import images -i eks-anywhere-images.tar
-eksctl anywhere copy packages --bundle ./eksa-bundle.yaml <private registry endpoint> --dst-cert rootCA.pem
 ```
+
+Download the EKS Anywhere bundle and use it to copy packages:
```

Comment:
If I start reading from line #73, it gets me confused of why I need to use the following commands to install packages. Wouldn't the download images and import images command suffice? Some clarification on the top description for this use case will be great.

Reply:
Okay, I added a sentence to summarize the process